alt-tab-macos/ui/Cell.swift

```diff
@@ -53,7 +53,7 @@ class Cell: NSCollectionViewItem {
         let fontIconWidth = Spaces.singleSpace && !openWindow!.isMinimized ? 0 : Preferences.fontIconSize + Preferences.interItemPadding
         label.textContainer!.size.width = thumbnail.frame.width - Preferences.iconSize! - Preferences.interItemPadding - fontIconWidth
         minimizedIcon.isHidden = !openWindow!.isMinimized
-        spaceIcon.isHidden = openWindow!.isMinimized || Spaces.singleSpace
+        spaceIcon.isHidden = openWindow!.isMinimized || Spaces.singleSpace || !Preferences.showSpaceNumberLabels
```

Comment:
I tested the PR again, and actually realized that this condition is not correct. If `Preferences.showSpaceNumberLabels` is `true`, which is the default, then `Spaces.singleSpace` is ignored, which is breaking the intent of not showing space labels for the single space use-case

I reflected on use-cases here. I thought we would need a dropdown menu like:

Show space number labels:
* Always
* Never
* If multiple spaces are used

However, I can't imagine anyone would want "Always". Why would someone want to see the space when they only use 1 space?

I think the only 3 useful use-cases are:

* I use 1 space at the moment: I don't need to see number labels since they are all the same
* I use multiple spaces at the moment: I prefer to see the labels to help me find which space a window belongs to
* I use multiple spaces at the moment: I prefer not seeing space labels; i'm fine

Thus I think the best UI here would be to rename the checkbox "Hide space number labels", and change the expression to:

```swift
spaceIcon.isHidden = openWindow!.isMinimized || Spaces.singleSpace || Preferences.hideSpaceNumberLabels
```

Reply:
Excellent points!

Done and done :)